websites/M/MKPC/presence.ts

```diff
@@ -57,5 +51,6 @@ presence.on("UpdateData", async () => {
     presence.setActivity();
   } else {
     presence.setActivity(presenceData);
+    presenceData.startTimestamp = browsingStamp;
```

Comment:
you are adding timestamp after setting activity? what?

Reply:
😂